documentation/views/asset-data.rst

```diff
@@ -7,39 +7,31 @@ Assets & sensor data
 Asset page
 ------------
 
-The asset page allows to plot data from the asset's sensors, show sensors and child assets and also to edit attributes of the asset, like its location.
+The asset page is divided into different sections. The default selection is the "Context" tab, which shows the structure and location of the asset.
```

Comment:
"On the context page, you can do the following:"

(then start a bulleted list of the four actions, that you mention)